### ISSUE TITLE
Limit code gallery image sizes

### ISSUE THREAD

**Issue by drwells** [dealii/dealii#5887]
When we generate HTML we need to provide bounds for some of the images. Users upload very high quality images (which is good) but they are too hard to see unless they are rescaled.

Example:

http://dealii.org/developer/doxygen/deal.II/code_gallery_Linear_Elastic_Active_Skeletal_Muscle_Model.html

The first two images are hundreds of times larger than the surrounding text in my web browser.

**Comment by jppelteret** [comment]
Yeah, you're right. Those pictures are epically large! I had no success at resizing them using markdown (at least, none of the options rendered in Atom). If its possible to do this in a generic way then that would be great. If not then I'll open a PR wherein I include the images with the HTML syntax.

**Comment by drwells** [comment]
My best idea is that (like what I recently did in step-49) we should 

```html
<img src="path/to/image.png" alt="" height="200px">
```
which will cause things to render in a restricted size.

I think that we already generate similar HTML as part of the code gallery build process, so (if I am correct) forcing everything to have a set limit might not be that hard.

**Comment by bangerth** [comment]
It's not the script that generates the doxygen input that is the problem here. Rather, for the case you link to above, it's the `readme.md` file:
```
...
The basic problem configuration, including a depiction of the underlying
muscle microstructural orientation, is (loosely) summarised in the following
image.
![Problem geometry](./doc/geometry/geometry.png)
...
```
That is, it needs to be fixed in the user submission. Do you know how to prescribe a size in markdown?

**Comment by jppelteret** [comment]
No, I don't (at least, I couldn't find a portable solution). There's no reference to image sizing on [Doxygen' markdown page](https://www.stack.nl/~dimitri/doxygen/manual/markdown.html#md_images). Should I just resize the images?

**Comment by jppelteret** [comment]
Or, as @drwells suggested I could include the images using HTML...
